cliutil/utils.go

```diff
@@ -9,40 +9,62 @@ import (
 	"runtime/debug"
 	"strings"
 
+	"github.com/ethereum/go-ethereum/common/hexutil"
 	ethcrypto "github.com/ethereum/go-ethereum/crypto"
 
 	"github.com/athanorlabs/atomic-swap/common"
 )
 
 var (
-	errNoEthereumPrivateKey = errors.New("must provide --ethereum-privkey file for non-development environment")
-	errInvalidEnv           = errors.New("--env must be one of mainnet, stagenet, or dev")
+	errInvalidEnv = errors.New("--env must be one of mainnet, stagenet, or dev")
 )
 
-// GetEthereumPrivateKey returns an ethereum private key for the given the CLI options.
+func createAndWriteEthKeyFile(ethPrivKeyFile string, env common.Environment, devXMRMaker, devXMRTaker bool) error {
+	var key *ecdsa.PrivateKey
+	var err error
+
+	switch {
+	case env == common.Development && devXMRMaker:
+		key, err = ethcrypto.HexToECDSA(common.DefaultPrivKeyXMRMaker)
+	case env == common.Development && devXMRTaker:
+		key, err = ethcrypto.HexToECDSA(common.DefaultPrivKeyXMRTaker)
+	default:
+		key, err = ethcrypto.GenerateKey()
+	}
```

Comment:
currently, both sides need to have a funded ETH key for the swap to work. once relayers are implemented, the xmr-maker won't need a funded ETH key, but the xmr-taker will still. we should probably add a log for this saying "fund this key"?

Reply:
I'll add a log

Reply:
Added a log, although I missed the word "fund". If you think we need it, feel free to suggest how to work it in.
```
New ETH wallet key generated in /home/dmitry/.atomicswap/alice-data-stagenet/eth.key
 with public address 0xD594c55bbBfdaC0b62c3C4C0f65533faaaeAA411
```

Reply:
I'd add either a bold log after this saying `this address must be funded to take an offer` or something like that, or I'd add a log before the balance check in `xmrtaker/net.go` (where `errBalanceTooLow` is returned) printing out the address again (or returning it in the error) saying "this account needs funds" or something along those lines

Reply:
Additional log added in both places.